modernism/static/_js/modernism.js

```diff
@@ -126,6 +126,24 @@ const setModalImage = (direction) => {
 
 }
 
+//Get the button
+let mybutton = document.getElementById("myBtn");
```

Comment:
Please rename the variable to sth for meaningful. 

Usually you want to create a name that is semantically explaining the type and/or the function the element fullfills, like here, it is a button that allows jumping to the top. You could find a word that describes this and use it as variable name and also as id in the the html context. 
Find here an [article](https://medium.com/wix-engineering/naming-convention-8-basic-rules-for-any-piece-of-code-c4c5f65b0c09) about that topic.

Reply:
I renamed the variable to "scrollToTopButton" because this is a better paraphrase for it and removed the comment.